src/Oscoin/Consensus/Nakamoto.hs

```diff
@@ -131,11 +131,18 @@ mineNakamoto
        , Hashable c (BlockHeader c (Sealed c PoW))
        )
     => Telemetry.Tracer m
-    -> (forall tx. [Block c tx (Sealed c PoW)] -> Telemetry.Traced Difficulty)
+    -> (forall tx. [Block c tx (Sealed c PoW)] -> Difficulty)
     -> Miner c PoW m
 mineNakamoto probed difiFn getBlocks unsealedBlock = do
     blks  <- getBlocks difficultyBlocks
-    diffy <- probed (difiFn blks)
+    let prevDifficulty = blockTargetDifficulty
```

Comment:
This suggests the difficulty is about to change - maybe `currentDifficulty` makes more sense

Reply:
Agreed.

Reply:
3bd77183